source/solvers/heat_transfer_assemblers.cc

```diff
@@ -355,6 +356,7 @@ HeatTransferAssemblerBDF<dim>::assemble_rhs(
 
               if (GGLS)
                 {
+                  // ?? why rho_cp * rho_cp
```

Comment:
same here

Reply:
You can look at the following paper for more info on GGLS
https://doi.org/10.1016/S0045-7825(02)00242-6